app/src/main/res/layout/material_drawer_header.xml

```diff
@@ -0,0 +1,134 @@
+<!-- this replaces the default material_drawer_header.xml from the MaterialDrawer library -->
```

Comment:
Are there any changes to this besides changing image classes?

Reply:
also added our placeholder as default